server/models/Meeting.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+
+const meetingSchema = new mongoose.Schema({
+  user_id: {
```

Comment:
To follow JavaScript convention, if this could be `userId` as opposed to snake case.